jira_test.go

```diff
@@ -777,38 +780,40 @@ func TestSetOptionMixFunc(t *testing.T) {
 	assert := assert.New(t)
 
 	// reset command line arg
-	oldArgs := os.Args
-	defer func() { os.Args = oldArgs }()
-
-	os.Args = append(os.Args, "--token=123")
-	os.Args = append(os.Args, "--type=license")
-	os.Args = append(os.Args, "--assigneeId=654")
-	os.Args = append(os.Args, "--api=http://snyk.io/api")
-	os.Args = append(os.Args, "--configFile=./fixtures")
+	args := []string{
```

Comment:
Arguments as they would be if we called `os.Args[1:]` (the first arg is the name of the tool)